src/domain/orders/details/swap/swap-details/shipping-methods.js

```diff
@@ -21,7 +21,7 @@ const SwapShippingMethods = ({ shipping_methods = [], taxRate, currency }) => {
                   <>
                     {method.shipping_option.name} -{" "}
                     {formatDecimalAndCurrency(
-                      ((100 + taxRate) * method.shipping_option.amount) / 10000,
```

Comment:
There is a util method in utils/prices.js for calculating this and checking for noDivisionCurrencies